css-images-4/Overview.bs

```diff
@@ -1941,7 +1941,7 @@ Color Stop “Fixup”</h4>
 	<pre class=prod>
 		&lt;image-1D> = <<stripes()>>
 		<<stripes()>> = stripes( <<color-stripe>># )
-		<<color-stripe>> = <<color>> && [ <<length-percentage>> | <<flex>> ]?
+		<<color-stripe>> = <<color>> && [ <<length-percentage>> | <<fraction>> ]?
```

Comment:
Unrelated to this PR but iirc, `0` et `∞` are always unambiguous and do not need a following unit, however `100` requires to be followed by `%` and `<length-percentage>` requires `[0,∞]`.

Simplified:

  > **`<length-percentage [0,∞]`**
  >
  > Negative values are invalid.
  >
  > Percentage thicknesses are relative to the *total width*.

Reply:
Never mind for the unitless `100`: it is unambiguous in `<percentage>`.

Reply:
Please open an issue so this can be discussed separately! FWIW, I think it's fine as it is. Values greater than 100% don't make much sense as a 100% wide color band already takes up the total width.